src/layout-card-editor.ts

```diff
@@ -44,8 +44,23 @@ class LayoutCardEditor extends LitElement {
     this._config = config;
   }
 
-  firstUpdated() {
+  async firstUpdated() {
     loadHaForm();
+    await this._preloadCardEditors();
+  }
+
+  async _preloadCardEditors() {
+    const helpers = await (window as any).loadCardHelpers?.();
+    if (!helpers) return;
+
+    const verticalStackCard = await helpers.createCardElement({
+      type: "vertical-stack",
+      cards: [],
+    });
+
+    if (verticalStackCard?.constructor?.getConfigElement) {
```

Comment:
Nice trick, but not quite enough.
You need to add an `await customElements.whenDefined("hui-vertical-stack-card")` before this `if`.

It can also be abbreviated as `await verticalStackCard?.constructor?.getConfigElement?.();`, but keeping the `if` makes it more readable.